xviz/v2/style.proto

```diff
@@ -24,8 +24,8 @@ message StyleClass {
 
 message StyleObjectValue {
   option (xviz_json_schema) = "style/object_value";
-  repeated uint32 fill_color = 1;
-  repeated uint32 stroke_color = 2;
+  bytes fill_color = 1;
+  bytes stroke_color = 2;
```

Comment:
Do you think we should change *all* color even the ones in the style to be the opaque byte ones?  Seems OK I guess I was just thinking "bulk color", but having it be consistent is probably better.

Reply:
yeah, I figure if we change one might as well change all, much simpler to treat it uniformly then.